pkgs/development/python-modules/rq/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchPypi, buildPythonPackage, mock, click, pytest, redis }:
+
+buildPythonPackage rec {
+  pname = "rq";
+  version = "1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rx8xji473ggmcy40v7lh0a69j4nrb4qrp4kp1766qg0swp37pi2";
+  };
+
+  checkInputs = [ pytest ];
```

Comment:
For me at least, this finds zero tests, and looking into this project's tests, they appear to heavily depend on having a running redis server, which is something we can't do. So you're probably better off adding `doCheck = false` and a comment of something like "tests require a running redis server".

Reply:
Oh, I see now... Updated according to your suggestion :pray: